mava/configs/logger/logger.yaml

```diff
@@ -1,23 +1,41 @@
 # --- Logging options ---
```

Comment:
A comment at the top of this config that explains how things work would be really nice.